src/components/data_comps/drof/cime_config/config_component.xml

```diff
@@ -13,12 +13,15 @@
   -->
 
   <description modifier_mode="1">
-    <desc rof="DROF[%NULL][%NYF][%IAF][%CPLHIST][%JRA]">Data runoff model</desc>
-    <desc option="NULL">NULL mode</desc>
-    <desc option="NYF" >COREv2 normal year forcing:</desc>
-    <desc option="IAF" >COREv2 interannual year forcing:</desc>
-    <desc option="CPLHIST">CPLHIST mode:</desc>
-    <desc option="JRA">JRA55 interannual forcing</desc>
+    <desc rof="DROF[%NULL][%NYF][%IAF][%IAFAIS00][%IAFAIS45][%IAFAIS55][%CPLHIST][%JRA]">Data runoff model</desc>
```

Comment:
Potential cesm impacts?

Reply:
I think it's okay as long as IAF and IAF* are not confused anywhere.